mappings/net/minecraft/util/math/Vec2f.mapping

```diff
@@ -1,7 +1,15 @@
 CLASS net/minecraft/class_241 net/minecraft/util/math/Vec2f
+	FIELD field_1336 SOUTH Lnet/minecraft/class_241;
+	FIELD field_1337 MAX Lnet/minecraft/class_241;
```

Comment:
Not sure what "max" means for a vector. If it's the vector `(Float.MAX_VALUE, Float.MAX_VALUE)`, then I think either `MAX_MAX` or `MAX_SOUTH_EAST` would be better.

Reply:
I’ll do MAX_SOUTH_EAST

Reply:
I’ll do MAX_SOUTH_EAST